_layouts/default.html

```diff
@@ -63,6 +63,60 @@
         <script defer src="/js/bootstrap.min.js"></script>
         <script defer src="/js/scripts.js"></script>
 
+        <!-- Schema.org JSON-LD Markup for Search Engine Optimization -->
+        <script type='application/ld+json'>
+        {
+            "@context": "https://schema.org/",
+            "@type": "SoftwareApplication",
+            "applicationCategory": "FinanceApplication",
+            "name": "Bisq",
+            "image": "https://bisq.network/images/bisq-logo.svg",
+            "availableOnDevice": [
+                "Desktop",
+                "Smartphone",
+                "Tablet"
+            ],
+            "author": {
+            "@type": "Organization",
```

Comment:
The Bisq DAO isn't really an organization...it's just a loose group of individuals who contribute to the software. I would prefer to see this `author` attribute removed, even if it comes at the expense of some clicks, since it's a legal grey area.

Reply:
> The Bisq DAO isn't really an organization...it's just a loose group of individuals who contribute to the software. I would prefer to see this `author` attribute removed, even if it comes at the expense of some clicks, since it's a legal grey area.

Ok, what about `Project` instead of `Organization`?

> An enterprise (potentially individual but typically collaborative), planned to achieve a particular aim. Use properties from Organization, subOrganization/parentOrganization to indicate project sub-structures.
([http://schema.org/Project](http://schema.org/Project))